translate/storage/lisa.py

```diff
@@ -18,6 +18,8 @@
 
 """Parent class for LISA standards (TMX, TBX, XLIFF)"""
 
+from xml.etree import ElementTree
```

Comment:
Missing import for the new code:

```suggestion
import io

from xml.etree import ElementTree
```